.travis.yml

```diff
@@ -11,8 +11,12 @@ env:
           use_coveralls=false
         - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
           testflags="--test-type integ_slow --pytest-args='-x'"
-        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev-nm-1.22-git
-          testflags="--test-type integ --pytest-args='-x'"
+        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
+          testflags="--test-type integ --pytest-args='-x'
+              --copr networkmanager/NetworkManager-1.22-git"
+        - CONTAINER_IMAGE=nmstate/centos8-nmstate-dev
+          testflags="--test-type integ --pytest-args='-x'
+              --copr networkmanager/NetworkManager-1.22-master"
```

Comment:
I'm surprised to see these as part of this commit. It is also not reasoned for and it is hard to understand "why".

This copr seems to be new or wrong: `networkmanager/NetworkManager-1.22-master`

Reply:
We remove that as we had rpmdb corruption. And we should test on these repo.

Reply:
And in this PR, we removed `centos8-nmstate-dev-nm-1.22-git` container image.